crates/settings/src/settings_content.rs

```diff
@@ -571,7 +571,6 @@ pub enum ModeContent {
     #[default]
     Normal,
     Insert,
-    HelixNormal,
```

Comment:
@kubkon I don't think it matters too much as helix mode is very clearly marked experimental; but we might want to add a serde alias to normal for this so that people's settings keep working.